pkgs/applications/misc/go-jira/default.nix

```diff
@@ -0,0 +1,22 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.2.1
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "go-jira-v1-${version}";
+  version = "1.0.17";
+  rev = "b98da3612d6ab0f3872b64ff36e6cf93219fb4d4";
+
+  goPackagePath = "gopkg.in/Netflix-Skunkworks/go-jira.v1";
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://gopkg.in/Netflix-Skunkworks/go-jira.v1";
+    sha256 = "0ap5dzikp934mawigmv7ighajr5yxpck2gpnb2cmkc1p6qa6gn4v";
+  };
+
+  meta = {
```

Comment:
Would you like to become the maintener of this packet? In this case, could you add yourself to the maintainers list?